validator/sawtooth_validator/concurrent/threadpool.py

```diff
@@ -88,31 +94,23 @@ def wrapper():
                 LOGGER.debug(
                     '(%s) Executing task %s', self._name, task_details)
 
-            return_value = None
-            try:
-                return_value = fn(*args, **kwargs)
-            # pylint: disable=broad-except
-            except Exception:
-                LOGGER.exception(
-                    '(%s) Unhandled exception during execution of task %s',
-                    self._name,
-                    task_details)
-
-            end_time = time.time()
-            run_time = (end_time - start_time) * 1000.0
+            with self._task_run_timer.time():
```

Comment:
This needs to be a timer per thread in the pool.  This measurement won't make sense otherwise.